WordPress/src/main/res/values/styles_toolbar.xml

```diff
@@ -49,6 +49,7 @@
 
     <style name="TextAppearance.App.CollapsingToolbar.Collapsed" parent="TextAppearance.MaterialComponents.Headline6">
         <item name="android:textColor">?attr/colorOnSurface</item>
+        <item name="fontFamily">sans-serif</item>
```

Comment:
Could you also add this to the two styles above, just to be on the safe side? Those would be `TextAppearance.App.Toolbar.Title` and `TextAppearance.App.CollapsingToolbar.Expanded`.

Reply:
Good idea! Done: 72a9967e3ea80302f667b6ffbe394c0c278be2ac